ivy_tests/test_ivy/test_frontends/test_torch/test_reduction_ops.py

```diff
@@ -1,12 +1,64 @@
 # global
 import numpy as np
 from hypothesis import given, strategies as st
+import pytest
 
 # local
 import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.test_functional.test_core.test_statistical import (
+    statistical_dtype_values,
+)
 from ivy_tests.test_ivy.helpers import handle_cmd_line_args
 
 
+def is_broadcastable(shp1, shp2):
+    for a, b in zip(shp1[::-1], shp2[::-1]):
+        if a == 1 or b == 1 or a == b:
+            pass
+        else:
+            return False
+    return True
+
+
+@pytest.mark.filterwarnings("ignore: dist")
```

Comment:
why was this added?

Reply:
Sorry ! this was by mistake, initially I was getting some warnings so I added this line to silence them.